### PR TITLE
Add metadata for the Fedora i3 Spin

### PR DESCRIPTION
**Pull request by D4N**
I have tried to add the Fedora i3 spin to all the places that I could find. Please tell me if anything is still missing.

### CODE REVIEW

**Comment by grulja** [comment]
You also want your metadata to be translatable:
https://github.com/FedoraQt/MediaWriter/blob/master/po/mediawriter.pot

**Comment by grulja** [comment]
We unfortunately don't have a way how to cleanly generate pot file from the metadata file. There is json2po utility, but it has to be manually cleaned so in case of adding a new spin, it's easier to add it manually.

**Comment by D4N** [comment]
Jan Grulich ***@***.***> writes:

> We unfortunately don't have a way how to cleanly generate pot file from the metadata file. There is json2po utility, but it has to be manually cleaned so in case of adding a new spin, it's easier to add it manually.

I've tried to create the pot file manually.

**Comment by grulja** [comment]
> Jan Grulich ***@***.***> writes:
> We unfortunately don't have a way how to cleanly generate pot file from the metadata file. There is json2po utility, but it has to be manually cleaned so in case of adding a new spin, it's easier to add it manually.
> I've tried to create the pot file manually.

Why don't you just add it at the end of the pot file? The order doesn't really matter here. That way you avoid bumping all numbers.

**Comment by grulja** [comment]
Hi, please check the current mediawriter.pot file and update your change accordingly. I think I fixed all translation issues and if you follow other spins, then your description should be properly translatable.

**Comment by D4N** [comment]
@grulja sorry for the long delay. I have rebased my branch on main and it looks like it's working:
mediawriter_i3

File: po/mediawriter.pot [hunk 1]
```diff
@@ -1251,6 +1251,42 @@ msgid ""
 "at all."
 msgstr ""
 
+#: [i3].subvariant
+msgctxt "Release|"
+msgid "i3"
+msgstr ""
+
+#: [i3].category
+msgctxt "Release|"
```
Comment (grulja) [inline po/mediawriter.pot:1260]: This has to be removed/empty, otherwise it will be a duplicated string to translate. See other spins.

**Comment by grulja** [comment]
I'm not sure about the logo. As you can clearly see it doesn't fit among other spins. We have all of them circles without colors.

**Comment by D4N** [comment]
> I'm not sure about the logo. As you can clearly see it doesn't fit among other spins. We have all of them circles without colors.

Indeed. I can try to modify it myself. @mairin could you then take a look at the end result (I know you're not the author of it, but I can't find Melody on Github)?

File: po/mediawriter.pot (hunk 1 above)
Comment (D4N) [reply po/mediawriter.pot:1260]: Sure, I will remove that.

**Comment by D4N** [comment]
I've tried my best to tweak the logo, it now looks like this:
mediawriter

File: po/mediawriter.pot (hunk 1 above)
Comment (D4N) [reply po/mediawriter.pot:1260]: I'm a bit confused here: the other spins also contain the line `msgctxt "Release|"`, so should it really be removed just for i3?

Comment (grulja) [reply po/mediawriter.pot:1260]: No, this one is category and it has to remain be the same.

**Comment by grulja** [comment]
> I've tried my best to tweak the logo, it now looks like this:
> mediawriter

Looks good to me.

File: po/mediawriter.pot (hunk 1 above)
Comment (grulja) [reply po/mediawriter.pot:1260]: Aha, it's still not removed.

Comment (D4N) [reply po/mediawriter.pot:1260]: I'm sorry, I really have no clue what I should do with the `.pot` file. Would you mind using github's suggestion feature to indicated what you mean or push directly to this PR?

Comment (grulja) [reply po/mediawriter.pot:1260]: Just remove:
```
msgctxt "Release|"
msgid "spins"
msgstr ""
```
and keep `#: [i3].category` comment.

Comment (D4N) [reply po/mediawriter.pot]: Apologies, now I get what you meant… Should be fixed now. Thanks for your patience!

**Pull request merged**